src/com/google/enterprise/adaptor/MetadataTransform.java

```diff
@@ -25,20 +27,111 @@
  */
 public interface MetadataTransform {
 
+  /** The key for the document's unique ID supplied in the {@code params}. */
   public static final String KEY_DOC_ID = "DocId";
+
+  /** The key for the document's display URL supplied in the {@code params}. */
   public static final String KEY_DISPLAY_URL = "Display-URL";
+
+  /** The key for the document's content type supplied in the {@code params}. */
   public static final String KEY_CONTENT_TYPE = "Content-Type";
+
+  /**
+   * The key for the document's crawl once flag supplied in the {@code params}.
+   */
   public static final String KEY_CRAWL_ONCE = "Crawl-Once";
+
+  /** The key for the document's lock flag supplied in the {@code params}. */
   public static final String KEY_LOCK = "Lock";
+
+  /**
+   * The key for the document's last modified timestamp supplied in the
+   * {@code params}.
+   */
   public static final String KEY_LAST_MODIFIED_MILLIS_UTC
       = "Last-Modified-Millis-UTC";
+
+  /**
+   * The key for a {@link TransmissionDecision} that may be added to the
+   * {@code params}.
+   */
   public static final String KEY_TRANSMISSION_DECISION 
       = "Transmission-Decision";
 
-  /** Transforms can cancel sending doc, or cancel sending its contents. */
+  /**
+   * Identifies the location a configured key/value pair resides,
+   * either in the {@link Metadata}, or the map of {@code params}
+   * supplied to the {@link #transform} method of a {@code MetadataTransform}.
```

Comment:
link here, but code for TransmissionDecision. Also, most of the time for a method link, you want to specify the text: s/#transform/#transform transform/